packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -92,24 +110,114 @@ class TickerMode extends StatefulWidget {
   ///
   /// In the absence of a [TickerMode] widget, this function returns a
   /// [ValueListenable], whose [ValueListenable.value] is always true.
+  @Deprecated(
+    'Use TickerMode.getValuesNotifier to get both enabled and forceFrames. '
+    'This feature was deprecated after v3.35.0-0.0.pre.',
+  )
   static ValueListenable<bool> getNotifier(BuildContext context) {
     final _EffectiveTickerMode? widget = context
         .getInheritedWidgetOfExactType<_EffectiveTickerMode>();
     return widget?.notifier ?? const _ConstantValueListenable<bool>(true);
   }
 
+  /// Returns the requested ticker mode values for this subtree and establishes
+  /// a dependency on the ancestor [TickerMode], if any.
+  ///
+  /// This is used automatically by [TickerProviderStateMixin] and
+  /// [SingleTickerProviderStateMixin] to decide if their tickers should be
+  /// enabled or disabled.
+  ///
+  /// In the absence of a [TickerMode] widget, this defaults to enabled
+  /// tickers that don't force frames.
+  ///
+  /// Typical usage is as follows:
+  ///
+  /// ```dart
+  /// bool tickingEnabled = TickerMode.valuesOf(context).enabled;
+  /// ```
+  static TickerModeData valuesOf(BuildContext context) {
+    final _EffectiveTickerMode? widget = context
+        .dependOnInheritedWidgetOfExactType<_EffectiveTickerMode>();
+    return widget?.values ?? TickerModeData.fallback;
+  }
+
+  /// Obtains a [ValueListenable] from the [TickerMode] surrounding the `context`,
+  /// which indicates whether tickers are enabled in the given subtree.
+  ///
+  /// When that [TickerMode] enabled or disabled tickers, the listenable notifies
+  /// its listeners.
```

Comment:
I think "enables or disables" would make more sense if I'm reading it right?

Reply:
"enables or disables" definitely sounds better